### ISSUE TITLE
Autosave

### ISSUE THREAD

**Issue by Gaurav0** [ember-cli/ember-twiddle#64]
But with an option to turn this off.

**Comment by joostdevries** [comment]
This sounds tricky to me. Isn't this more about making the `Save` action more accessible?

**Comment by sheriffderek** [comment]
I agree that it should be _conscious_, and having it more available, à la jsfiddle, in the header. This may also solve #194. Localstorage sound like a really nice "Doh!" buffer to add as well. Too many people auto persisting at a high rate can't be great.

**Comment by cromwellryan** [comment]
RE _auto persisting at a high rate can't be great._ and _saving to persistence should be a conscious action._

why @sheriffderek and @joostdevries?

**Comment by sheriffderek** [comment]
@cromwellryan One reason would be that your gist's revision history would grow very quickly and then become useless and quickly up into the hundreds if not thousands. I like to use twiddle to show short walk-throughs of Ember things - and I like to save at each important step of the process. Then the gist has the history of each addition to complete the example. I think that the local storage idea would be great.

I'm not sure what the details are on the github API - but with most APIs I've worked with have max request limits. I would imagine if thousands of us were using twiddle and autosaving every 30 seconds that our current _arrangement_ with github may need to be reassessed. But I don't know the facts behind that. I would be interested to know.
